ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Query.hs

```diff
@@ -65,12 +79,19 @@ data Query blk result where
   -- Supported by 'QueryVersion' >= 'QueryVersion1'.
   GetSystemStart :: Query blk SystemStart
 
+  -- | Get the 'GetHeaderStateTip' time.
+  --
+  -- Supported by 'QueryVersion' >= 'QueryVersionX'. -- TODO jky what version?
+  GetHeaderStateTip :: Query blk (WithOrigin (HeaderStateTip blk))
```

Comment:
@DavidEichmann Do you know if I can use `QueryVersion1` or do I need to use `QueryVersion2`?

Reply:
Hey @newhoggy. You are indeed augmenting the protocol with a new query, so the version must be bumped :+1: .

Reply:
... @newhoggy  I perhaps spoke too quickly. There is an exception. You don't need to bump the version number if the previous version number has not yet been released, so you should confirm that. Specifically, figure out what commit was the latest release of `ouroboros-network` (I assume this corresponded to the release of Alonzo). If `QueryVersion1` was added after that release, then you actually don't need to bump the version number.

Reply:
@nfrisby Is there some procedure for easily knowing what the latest release commit hash is? I admit I'm not familiar with the release process.

Reply:
@DavidEichmann I go to the "Releases" page of `cardano-node` to find the commit of the latest release, and look at the `ouroboros-network` commit hash listed in the corresponding `cabal.project` file. I think that's the source of truth.